packages/apps/reputation-oracle/server/src/modules/webhook/webhook-incoming.entity.ts

```diff
@@ -2,7 +2,7 @@ import { Column, Entity } from 'typeorm';
 
 import { NS } from '../../common/constants';
 import { BaseEntity } from '../../database/base.entity';
-import { WebhookStatus } from '../../common/enums';
+import { EventType, WebhookStatus } from '../../common/enums';
```

Comment:
EventType should be added to the WebhookIncomingEntity?

Reply:
At the moment I don't think we need it because it's just one type of event and we don't plan to have more.